src/patternfly/components/Drawer/drawer.scss

```
@@ -443,12 +445,9 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
     }
 
     > .pf-c-drawer__main > .pf-c-drawer__panel.pf-m-resizable {
-      padding-right: var(--pf-c-drawer--m-panel-left__panel--m-resizable--PaddingRight);
-      padding-left: 0;
-
       > .pf-c-drawer__splitter {
-        --pf-c-drawer__splitter--Right: 0;
-        --pf-c-drawer__splitter--Left: auto;
+        order: 2;
```

We try to use the first value needed to make the change, so `order: 1;` should work since nothing else in this flex grouping has an `order` 

```suggestion
        order: 1;
```